setup.md

```diff
@@ -2,4 +2,54 @@
 title: Setup
 ---
 
-#FIXME
+## Setup Python
+
+Make sure that you have the latest Python installed. The latest Python released is `Python 3.11.3`
+
+- To check the installation open a terminal and run:
+
+```bash
+python –version
+```
+- or if you have Python3:
```

Comment:
Python 3.x is required for this training, and PEP 394 requires all Python 3.x to provide `python3`, so I'd just write the test for only that command.